app/models/GoodsTypeModel.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.i18n.Messages
+
+
+object GoodsTypeModel {
```

Comment:
![Screenshot 2023-11-07 at 18 57 36](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/9093361/a9b5df6e-28fc-4d68-9885-d3f7c2129c42)
Missing fermeremented beverage?

Reply:
not neeeded , fermented beverage included in wine